### ISSUE TITLE
[BUG] Celery beat crashing due to wrong db file name

### ISSUE THREAD

**Issue by SpiderD555** [paperless-ngx/paperless-ngx#7823]
### Description

Celery beat for Paperless is not working on 2.12.1. It crashes immediately after I try to start it.
It worked fine on 2.10.2 (before I made upgrade).

This is bare metal installation on VM

Data folder looks like this, so I guess the filename is the issue:
root@paperless1:/opt/paperless/paperless-v2.12.1/paperless-ngx# ls -la /opt/paperless/data/
total 28
drwxr-x--- 4 paperless paperless  4096 Oct  1 13:42 .
drwxr-x--- 8 paperless paperless  4096 Sep 29 11:06 ..
-rw-r--r-- 1 paperless paperless 12288 Oct  1 13:43 **celerybeat-schedule.db.db**
drwxr-xr-x 2 paperless paperless  4096 Oct  1 13:33 index
drwxr-xr-x 2 paperless paperless  4096 Sep 30 01:31 log

For me this is a clear duplicate of https://github.com/paperless-ngx/paperless-ngx/issues/3739

Reboots didn't help. Only after renaming the file it all started to work again. I would like to leave this information here, so other people having the same issue will be able to fix it. Not sure why the file got the strange name in the first place.
Feel free to close the case, since I managed to fix it, but you may want to consider checking the source code since I am not the only one experiencing this.

Here's the log:
```
Oct 01 13:31:58 paperless1 systemd[1]: Started paperless-scheduler.service - Paperless Celery Beat.
Oct 01 13:31:59 paperless1 celery[2860]: [2024-10-01 13:31:59,995] [INFO] [celery.beat] beat: Starting...
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,003] [CRITICAL] [celery.beat] beat raised exception <class '_dbm.error'>: error('cannot add item to database')
Oct 01 13:32:00 paperless1 celery[2860]: Traceback (most recent call last):
Oct 01 13:32:00 paperless1 celery[2860]:   File "/usr/lib/python3.11/shelve.py", line 111, in __getitem__
Oct 01 13:32:00 paperless1 celery[2860]:     value = self.cache[key]
Oct 01 13:32:00 paperless1 celery[2860]:             ~~~~~~~~~~^^^^^
Oct 01 13:32:00 paperless1 celery[2860]: KeyError: 'entries'
Oct 01 13:32:00 paperless1 celery[2860]: During handling of the above exception, another exception occurred:
Oct 01 13:32:00 paperless1 celery[2860]: Traceback (most recent call last):
Oct 01 13:32:00 paperless1 celery[2860]:   File "/opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/celery/beat.py", line 570, in _create_schedule
Oct 01 13:32:00 paperless1 celery[2860]:     self._store['entries']
Oct 01 13:32:00 paperless1 celery[2860]:     ~~~~~~~~~~~^^^^^^^^^^^
Oct 01 13:32:00 paperless1 celery[2860]:   File "/usr/lib/python3.11/shelve.py", line 113, in __getitem__
Oct 01 13:32:00 paperless1 celery[2860]:     f = BytesIO(self.dict[key.encode(self.keyencoding)])
Oct 01 13:32:00 paperless1 celery[2860]:                 ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Oct 01 13:32:00 paperless1 celery[2860]: KeyError: b'entries'
Oct 01 13:32:00 paperless1 celery[2860]: During handling of the above exception, another exception occurred:
Oct 01 13:32:00 paperless1 celery[2860]: Traceback (most recent call last):
Oct 01 13:32:00 paperless1 celery[2860]:   File "/opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/celery/apps/beat.py", line 113, in start_scheduler
Oct 01 13:32:00 paperless1 celery[2860]:     service.start()
Oct 01 13:32:00 paperless1 celery[2860]:   File "/opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/celery/beat.py", line 634, in start
Oct 01 13:32:00 paperless1 celery[2860]:     humanize_seconds(self.scheduler.max_interval))
Oct 01 13:32:00 paperless1 celery[2860]:                      ^^^^^^^^^^^^^^
Oct 01 13:32:00 paperless1 celery[2860]:   File "/opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/kombu/utils/objects.py", line 40, in __get__
Oct 01 13:32:00 paperless1 celery[2860]:     return super().__get__(instance, owner)
Oct 01 13:32:00 paperless1 celery[2860]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Oct 01 13:32:00 paperless1 celery[2860]:   File "/usr/lib/python3.11/functools.py", line 1001, in __get__
Oct 01 13:32:00 paperless1 celery[2860]:     val = self.func(instance)
Oct 01 13:32:00 paperless1 celery[2860]:           ^^^^^^^^^^^^^^^^^^^
Oct 01 13:32:00 paperless1 celery[2860]:   File "/opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/celery/beat.py", line 677, in scheduler
Oct 01 13:32:00 paperless1 celery[2860]:     return self.get_scheduler()
Oct 01 13:32:00 paperless1 celery[2860]:            ^^^^^^^^^^^^^^^^^^^^
Oct 01 13:32:00 paperless1 celery[2860]:   File "/opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/celery/beat.py", line 668, in get_scheduler
Oct 01 13:32:00 paperless1 celery[2860]:     return symbol_by_name(self.scheduler_cls, aliases=aliases)(
Oct 01 13:32:00 paperless1 celery[2860]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Oct 01 13:32:00 paperless1 celery[2860]:   File "/opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/celery/beat.py", line 513, in __init__
Oct 01 13:32:00 paperless1 celery[2860]:     super().__init__(*args, **kwargs)
Oct 01 13:32:00 paperless1 celery[2860]:   File "/opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/celery/beat.py", line 264, in __init__
Oct 01 13:32:00 paperless1 celery[2860]:     self.setup_schedule()
Oct 01 13:32:00 paperless1 celery[2860]:   File "/opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/celery/beat.py", line 541, in setup_schedule
Oct 01 13:32:00 paperless1 celery[2860]:     self._create_schedule()
Oct 01 13:32:00 paperless1 celery[2860]:   File "/opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/celery/beat.py", line 574, in _create_schedule
Oct 01 13:32:00 paperless1 celery[2860]:     self._store['entries'] = {}
Oct 01 13:32:00 paperless1 celery[2860]:     ~~~~~~~~~~~^^^^^^^^^^^
Oct 01 13:32:00 paperless1 celery[2860]:   File "/usr/lib/python3.11/shelve.py", line 125, in __setitem__
Oct 01 13:32:00 paperless1 celery[2860]:     self.dict[key.encode(self.keyencoding)] = f.getvalue()
Oct 01 13:32:00 paperless1 celery[2860]:     ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Oct 01 13:32:00 paperless1 celery[2860]: _dbm.error: cannot add item to database
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,005] [WARNING] [celery.redirected] Traceback (most recent call last):
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,005] [WARNING] [celery.redirected]   File "/usr/lib/python3.11/shelve.py", line 111, in __getitem__
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,005] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,005] [WARNING] [celery.redirected] value = self.cache[key]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,005] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,005] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,006] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,006] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,006] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,006] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,006] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,006] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,006] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,006] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,007] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,007] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,007] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,007] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,007] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,007] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,007] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,007] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,008] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,008] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,008] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,008] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,008] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,008] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,008] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,008] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,009] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,009] [WARNING] [celery.redirected] KeyError
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,009] [WARNING] [celery.redirected] :
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,009] [WARNING] [celery.redirected] 'entries'
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,009] [WARNING] [celery.redirected] During handling of the above exception, another exception occurred:
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,009] [WARNING] [celery.redirected] Traceback (most recent call last):
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,009] [WARNING] [celery.redirected]   File "/opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/celery/beat.py", line 570, in _create_schedule
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,010] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,010] [WARNING] [celery.redirected] self._store['entries']
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,010] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,010] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,010] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,010] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,010] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,010] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,011] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,011] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,011] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,011] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,011] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,011] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,011] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,011] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,012] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,012] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,012] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,012] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,012] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,012] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,012] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,012] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,013] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,013] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,013] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,013] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,013] [WARNING] [celery.redirected]   File "/usr/lib/python3.11/shelve.py", line 113, in __getitem__
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,013] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,013] [WARNING] [celery.redirected] f = BytesIO(self.dict[key.encode(self.keyencoding)])
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,013] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,014] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,014] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,014] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,014] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,014] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,014] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,014] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,014] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,015] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,015] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,015] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,015] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,015] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,015] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,015] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,015] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,015] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,016] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,016] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,016] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,016] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,016] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,016] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,016] [WARNING] [celery.redirected] ~
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,016] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,017] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,017] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,017] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,017] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,017] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,017] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,017] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,017] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,018] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,018] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,018] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,018] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,018] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,018] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,018] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,018] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,018] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,019] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,019] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,019] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,019] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,019] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,019] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,019] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,019] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,020] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,020] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,020] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,020] [WARNING] [celery.redirected] ^
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,020] [WARNING] [celery.redirected] KeyError
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,020] [WARNING] [celery.redirected] :
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,020] [WARNING] [celery.redirected] b'entries'
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,020] [WARNING] [celery.redirected] During handling of the above exception, another exception occurred:
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,021] [WARNING] [celery.redirected] Traceback (most recent call last):
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,021] [WARNING] [celery.redirected]   File "/opt/paperless/paperless-v2.12.1/venv/bin/celery", line 8, in <module>
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,021] [WARNING] [celery.redirected]
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,021] [WARNING] [celery.redirected] sys.exit(main())
Oct 01 13:32:00 paperless1 celery[2860]: [2024-10-01 13:32:00,021] [WARNING] [celery.redirected]

====
cut excess logs
====

Oct 01 13:32:00 paperless1 celery[2860]: celery beat v5.4.0 (opalescent) is starting.
Oct 01 13:32:00 paperless1 celery[2860]: __    -    ... __   -        _
Oct 01 13:32:00 paperless1 celery[2860]: LocalTime -> 2024-10-01 13:31:59
Oct 01 13:32:00 paperless1 celery[2860]: Configuration ->
Oct 01 13:32:00 paperless1 celery[2860]:     . broker -> redis://localhost:6379//
Oct 01 13:32:00 paperless1 celery[2860]:     . loader -> celery.loaders.app.AppLoader
Oct 01 13:32:00 paperless1 celery[2860]:     . scheduler -> celery.beat.PersistentScheduler
Oct 01 13:32:00 paperless1 celery[2860]:     . db -> /opt/paperless/data/celerybeat-schedule.db
Oct 01 13:32:00 paperless1 celery[2860]:     . logfile -> [stderr]@%INFO
Oct 01 13:32:00 paperless1 celery[2860]:     . maxinterval -> 5.00 minutes (300s)
Oct 01 13:32:00 paperless1 systemd[1]: paperless-scheduler.service: Main process exited, code=exited, status=1/FAILURE
Oct 01 13:32:00 paperless1 systemd[1]: paperless-scheduler.service: Failed with result 'exit-code'.
Oct 01 13:32:00 paperless1 systemd[1]: paperless-scheduler.service: Consumed 2.448s CPU time.
```

### Steps to reproduce

1. Upgrade paperless from 2.10.2 to 2.12.1
2. Paperless starts showing bad behavior - Some error message on main page reffering to docs or just plain 504 error from reverse proxy.

### Webserver logs

```bash
Oct 01 13:45:30 paperless1 gunicorn[639]: /opt/paperless/paperless-v2.12.1/venv/lib/python3.11/site-packages/django/http/response.py:517: Warning: StreamingHttpResponse must consume synchronous iterators in order to serve them asynchrono>
Oct 01 13:45:30 paperless1 gunicorn[639]:   warnings.warn(
```

### Paperless-ngx version

2.12.1

### Host OS

Debian 12

### Installation method

Docker - official image

### System status

```json
I don't see any System Status tab
```

### Browser

Vivaldi

### Configuration changes

Yes, but nothing outstanding. I have set my own url, language, OIDC integration and OCR arguments

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [X] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

**Comment by stumpylog** [comment]
That's not the filename, it's a corrupt file. Simply remove and restart.

**Comment by SpiderD555** [comment]
@stumpylog Today I have encountered the bug again.
Deleting the db file helps but only for a week or so.
```
root@paperless1:/opt/paperless/data# ls
celerybeat-schedule.db.db  classification_model.pickle	index  log
```

**Comment by shamoon** [comment]
https://github.com/celery/celery/pull/9331 by our own stumpylog was recently merged which should help, when its released we can pull that into paperless-ngx
